media/css/protocol/components/_footer.scss

```
@@ -177,14 +184,38 @@ $image-path: '/media/protocol/img';
     }
 }
 
+.mzp-c-footer-primary {
+    @media screen and (min-width: #{$screen-lg}) {
+        @include footer-line;
+        position: relative;
+    }
+}
+
 .c-footer-logo {
     @extend .mzp-c-footer-primary-logo; /* stylelint-disable-line scss/at-extend-no-missing-placeholder */
+    text-align: center;
+
+    @media screen and (min-width: #{$screen-lg}) {
+        position: absolute;
+        bottom: 0;
+        right: 0;
```

To fix https://github.com/mozmeao/springfield/pull/278#discussion_r2140955358 just tested with:

```suggestion
        @include bidi(((right, 0, left, auto),));
```